tensorflow/core/common_runtime/pluggable_device/BUILD

```diff
@@ -69,18 +76,50 @@ cc_library(
     copts = tf_copts(),
     visibility = ["//visibility:public"],
     deps = [
-        ":pluggable_device_runtime_impl",
         "//tensorflow/c/experimental/stream_executor",
-    ],
+        "//tensorflow/c/experimental/stream_executor:stream_executor_internal",
+        "//tensorflow/core:core_cpu",
+        "//tensorflow/core:framework",
+        "//tensorflow/core:lib",
+        "//tensorflow/core:lib_internal",
+        "//tensorflow/core:protos_all_cc",
+        "//tensorflow/core/common_runtime:bfc_allocator",
+        "//tensorflow/core/common_runtime:dma_helper",
+        "//tensorflow/core/common_runtime:local_device",
+        "//tensorflow/core/common_runtime:process_state",
+        "//tensorflow/core/common_runtime:shared_counter",
+        "//tensorflow/core/platform:stream_executor",
+        "//tensorflow/stream_executor:event",
+        "//tensorflow/stream_executor:kernel",
+    ] + if_static([
```

Comment:
@jzhoulon and @penpornk  
This MacOS test failure commit in pluggable impl seems to have caused regression on mac platforms to register pluggable device.  When _pywrap_tensorflow .so loads up the plugin it goes through the device initialization fine then it hands over to libtensorflow_framework dylib to query the plugin handle using the Platform name in  `MultiPlatformManager` . And during this part it fails with "Platform " not found. Seems like a linker issue where the plugin registry is not getting shared across the `so` and `dylib`. 

Reverting this locally workarounds it.  Will create a PR with proper fix.